content/en/monitors/_index.md

```diff
@@ -36,91 +30,58 @@ cascade:
+## Getting started
 
-To create a monitor programmatically, see the [Datadog API][7] or [community maintained libraries][8].
+The fastest way to start with Datadog Monitors is with [Recommended Monitors][1]. These are a collection of monitors within Datadog that are preconfigured by Datadog and integration partners.
 
-## Configure notifications and automations
+You can also build your own monitors from scratch in lab environments in the Learning Center, or in your application by following the Getting Started with Monitors guide.
 
-{{< img src="/monitors/notify.png" alt="Notify when a monitor is alerting" style="width:90%;">}}
-
-Set up [Monitor Notifications][11] when creating monitors to keep your team informed of issues. Route the notifications to the correct people,  include [workflow automations][17], [cases][18], and [Datadog team handles][19], leverage template variables to include details, and attach snapshots when sending the alerts by email or Slack. Create [downtimes][12] to mute alerts during application maintenance.
-
-## Manage monitors
-
-{{< img src="/monitors/manage.png" alt="Manage all monitors alerts" style="width:90%;">}}
-
-[Manage Monitors][13] by editing, cloning, deleting, muting, and resolving monitors all in the same place. Focus on high priority alerts by using advanced faceted search. Explore monitor details and alerts over time on the [Monitors List page][9].
-
-## Export and import monitors
-
-To export a monitor:
-
-1. From the [**Manage Monitors**][9] page, click the monitor you want to export.
-1. You should see the Monitor Status page.
-1. Click the settings cog (top right) and select **Export** from the menu.
+{{< whatsnext desc="Use the following resources to create:" >}}
+    {{< nextlink href="/getting_started/monitors/" >}}Getting started with Monitors: Guide on how to build a metric based monitor{{< /nextlink >}}
+    {{< nextlink href="/monitors/types/" >}}Create a monitor from Monitor Types{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/datadog-foundation" >}}Learning Center: Build a metric base monitor in a sandbox lab environment{{< /nextlink >}}
+{{< /whatsnext >}}
 
-To import a monitor:
+## Analyze aggregate data
 
-1. Navigate to [**Monitors** > **New Monitor**][4].
-1. Click [**Import from JSON**][10] at the top of the page.
-1. Add your JSON monitor definition and click **Save**.
+Good data should be well-understood, granular, tagged by scope, and long-lived. Use different data types for alerts and diagnostics, based on the level of urgency. Instrument everything and collect as much relevant data as possible for comprehensive measurements and observability of complex systems.
 
-## Control monitor tags with tag policies
+Measure the health of your applications and the state of your infrastructure through data sent to Datadog. Use data from across the platform whether Logs, Metrics, Traces, Cloud Cost, RUM, or Security, to create alerts on the potential issues.
```

Comment:
Can't decide if better to keep the list of Datadog platform components. Defer to you!

```suggestion
Measure the health of your applications and the state of your infrastructure with Datadog. Use data from across the Datadog platform to create alerts on potential issues.
```